controllers/api/openstack/templatelist.py

```diff
@@ -18,18 +19,21 @@ def GET(self):
         auth_string = cherrypy.request.cookie.get('session').value
 
         novaClient = getNovaInstance()
```

Comment:
Is nova still used somewhere or has it been completely replaced by glance?

Reply:
Yes, the nova is still the primary api but it no longer proxies requests to glance.